pkg/plugins/manager/pipeline/bootstrap/steps.go

```diff
@@ -153,3 +155,15 @@ func configureAppChildPlugin(cfg *config.Cfg, parent *plugins.Plugin, child *plu
 		child.Module = path.Join("/", cfg.GrafanaAppSubURL, "/public/plugins", parent.ID, appSubPath, "module.js")
 	}
 }
+
+// SkipHostEnvVarsDecorateFunc returns a DecorateFunc that configures the SkipHostEnvVars field of the plugin.
+// It will be set to true if the FlagPluginsSkipHostEnvVars feature flag is set, and the plugin does not have
+// skip_host_env_vars = false in its plugin settings.
+func SkipHostEnvVarsDecorateFunc(cfg *config.Cfg) DecorateFunc {
+	return func(_ context.Context, p *plugins.Plugin) (*plugins.Plugin, error) {
+		p.SkipHostEnvVars = cfg.Features != nil &&
+			cfg.Features.IsEnabledGlobally(featuremgmt.FlagPluginsSkipHostEnvVars) &&
+			cfg.PluginSettings[p.ID]["skip_host_env_vars"] != "false"
```

Comment:
Nit. I find it a bit confusing/non-optimal that `skip_host_env_vars` by default is true/empty. Can we change the setting name so default is false/empty, e.g. `forward_host_env_vars`?

Reply:
Sure, I have changed it :smile: